controllers/handlers.go

```diff
@@ -108,6 +110,10 @@ func HandleUpload(uploader TokenUploader) func(http.ResponseWriter, *http.Reques
 		}
 
 		if err := uploader.Upload(ctx, tokenObjectName, tokenObjectNamespace, data); err != nil {
+			if unwrap := errors.Unwrap(err); kubeErrors.IsNotFound(unwrap) {
```

Comment:
Nice! I wonder if we should not react the same way for 403 - forbidden and possibly others here, too?